devel/libptytty/Portfile

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                libptytty
```

Comment:
It's confusing to have your PR called "rxvt-unicode: ..." which adds another completely unrelated port to the PR.

It would make sense to split the other one into its own separate PR.